addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5055,6 +5061,30 @@ registry.layout_column = SnippetOptionWidget.extend({
         gridUtils._resizeGrid(rowEl);
         this.trigger_up('activate_snippet', {$snippet: $(newColumnEl)});
     },
+    /**
+     * @override
+     * @param {boolean} previewMode - @see this.selectClass
+     * @param {string} widgetValue
+     * @param {Object} params
+     * @returns {Promise|undefined}
+     */
+    selectStyle(previewMode, widgetValue, params) {
+        this._super(...arguments);
```

Comment:
Either return `this._super(...arguments)` or turn your function into an async one and await it at the start.
Not doing so could lead to race conditions where `this._super(...arguments)` returns a promise that's not awaited by anything.